### DISCUSSION TITLE
Instant total scores in feedback

### DISCUSSION THREAD

**Discussion by erisweety** [jspsych/jsPsych#828] [Q&A] [answered]
Hi, I am wondering if there is a plugin for displaying an instant total score as part of the feedback after a trial. For example, in a task with 100 trials, the participant has got 20 correct when s/he reaches the 40th, and 50% would be shown on the screen as part of the feedback, so that the participant knows how well s/he is doing. Thank you!

**Comment by jkhartshorne** [comment, marked as answer]
There's actually an example in the demo on the jsPsych website! Check it out [here](https://www.jspsych.org/tutorials/rt-task/). If you want to look for specifically the relevant code, search for 'correct_trials'.
